### PR TITLE
15.0 luxer copy property from subscription to invoice delivery address 2874625 romf

### PR DESCRIPTION
**Pull request by romf-odoo**
### Description

*Luxer: Copy property from subscription to invoice delivery address*

Link to task: https://www.odoo.com/web#id=2874640&cids=17&menu_id=4720&action=4665&active_id=2874625&model=project.task&view_type=form

### CODE REVIEW

File: luxer/views/sale_order_inherit.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_form_inherit_order" model="ir.ui.view">
```
Comment (jefe-odoo) [inline luxer/views/sale_order_inherit.xml:4]: use parent ID

File: luxer/views/sale_order_inherit.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            <field name="name">sale.order.inherit</field>
```
Comment (jefe-odoo) [inline luxer/views/sale_order_inherit.xml:5]: Add module name after inherit e.g. `sale.order.inherit.luxer`

File: luxer/views/subscription_inherit.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_form_inherit_subscription" model="ir.ui.view">
```
Comment (jefe-odoo) [inline luxer/views/subscription_inherit.xml:4]: Use parent ID

File: luxer/models/luxer.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    def _prepare_invoice_data(self):
+        self.ensure_one()
+        vals = super()._prepare_invoice_data()
+        vals['partner_shipping_id'] = self.property_partner.id or vals.get(
+            'partner_shipping_id')
+        return vals
+
+    def _prepare_renewal_order_values(self, discard_product_ids=False, new_lines_ids=False):
+        res = super()._prepare_renewal_order_values(discard_product_ids, new_lines_ids)
+        for subscription in self.filtered('property_partner'):
```
[the remark this answers is not in the document]
Comment (romf-odoo) [reply luxer/models/luxer.py:22]: Yes there are! but as the delivery address is being set to the `res.partner`  if contact info is set and if not then it takes the customer address so it always contains a value so I don't think it should be an issue

**Pull request closed without merging**